README.md

```diff
@@ -45,15 +50,32 @@ Note that:
   a shifting element.
 
 * An element whose start location changes by less than 3 CSS pixels is not a
-  shifting element.
+  shifting element.  This threshold was chosen to avoid penalizing trivial
+  movements.  It also allows some animations (but a broader allowance is
+  described in "Transform Changes" below).
```

Comment:
Can this have a link?

Reply:
I think since "Transform Changes" is literally the next line it is more confusing to have a link which will change the scroll position